spec/models/measure_condition_dialog_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'spec_helper'
+
+describe MeasureConditionDialog do
+  subject(:dialog) { described_class.build(declarable, measure) }
+
+  before do
+    stub_const('MeasureConditionDialog::CONFIG_FILE_NAME',
+               File.join('spec', 'fixtures', 'measure_condition_dialog_config.yaml'))
+  end
```

Comment:
There was still a problem: our tests were linked to real data,
in fact, when I commented out 2 entries the tests failed.
So I created a fixture file just for testing.